### DISCUSSION TITLE
Question about the createElement function

### DISCUSSION THREAD

**Discussion by danielleroux** [remarkablemark/html-react-parser#1649] [General]
I discovered a strange behavior of the parser function. If I use the parser function within a library (stencil-output-target/react) I get a different $$type property then within a nextjs project.

Inside the react library '$$typeof': Symbol(react.element) and inside nextjs 
$$typeof: Symbol(react.transitional.element) 

React 19 and nextjs 15. Does anybody have a idea how this could happen?

**Comment by remarkablemark** [comment]
Can you provide a reproducible example? E.g., [StackBlitz](https://stackblitz.com/edit/html-react-parser-typescript)
